src/cgame/cg_tutorial.cpp

```diff
@@ -545,15 +549,13 @@ static void CG_SpectatorText( char *text, playerState_t *ps )
 		else if ( cgs.clientinfo[ cg.clientNum ].team == TEAM_NONE )
```

Comment:
I think this should just be `else {` ? Since you can switch camera type regardless of whether you are on the spectator team.

Reply:
done